Doc/reference/datamodel.rst

```diff
@@ -804,16 +809,34 @@ Built-in functions
 A built-in function object is a wrapper around a C function.  Examples of
 built-in functions are :func:`len` and :func:`math.sin` (:mod:`math` is a
 standard built-in module). The number and type of the arguments are
-determined by the C function. Special read-only attributes:
+determined by the C function.
+
+Special read-only attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+
+   * - Attribute
+     - Meaning
+
+   * - .. attribute:: function.__doc__
```

Comment:
I find it a little confusing that this attribute (for builtin functions specifically) is labelled as `function.__doc__`, but for information on this attribute, the entry links to another attribute (for functions in general), which is also labelled as `function.__doc__`.

Maybe this attribute should be called `builtin_function.__doc__`? (And similarly for the other attributes on builtin functions?) That would have the added benefit that Sphinx would generate an anchor for this entry, that people could link directly to

Reply:
Done.